saas_app/locale/fr/LC_MESSAGES/django.po

```diff
@@ -169,89 +169,97 @@ msgid "There was an error sending the email to the S32 approver."
 
-#: internal_ops/views.py:156
+#: internal_ops/views.py:157
 msgid "The form was successfully saved."
 msgstr "Le formulaire a été enregistré avec succès."
 
-#: internal_ops/views.py:160
+#: internal_ops/views.py:161
 msgid "There was an error saving the form."
 msgstr "Le formulaire a été enregistré avec succès"
 
-#: internal_ops/views.py:164
+#: internal_ops/views.py:165
 msgid "Please add the fund center and the approver."
 msgstr "Veuillez ajouter le centre financier et l'approbateur."
 
-#: internal_ops/views.py:172
+#: internal_ops/views.py:173
 msgid "Sent to S32 Approver for Approval"
 msgstr "Envoyé à l'approbateur S32 pour approbation"
 
-#: internal_ops/views.py:181
+#: internal_ops/views.py:182
 msgid "There was an error updating the status of the form."
 msgstr ""
 "Une erreur s'est produite lors de la mise à jour du statut du formulaire"
 
-#: internal_ops/views.py:189
+#: internal_ops/views.py:190
 msgid "Please add the fund center and the approver and then save the form."
 msgstr ""
 "Veuillez ajouter le centre financier et l'approbateur, puis enregistrer le "
 "formulaire."
 
-#: internal_ops/views.py:208
+#: internal_ops/views.py:209
 msgid "We have successfully emailed the S32 approver."
 msgstr "Nous avons réussi à envoyer un e-mail à l'approbateur S32."
 
-#: internal_ops/views.py:215
+#: internal_ops/views.py:216
 msgid ""
 "There was an error sending the notification emails to the S32 approver or "
 "requestor."
 msgstr ""
 "Une erreur s'est produite lors de l'envoi des e-mails de notification à "
 "l'approbateur ou au demandeur S32"
 
-#: internal_ops/views.py:238
+#: internal_ops/views.py:239
 msgid "We have successfully emailed the requestor."
 msgstr "Nous avons réussi à envoyer un e-mail au demandeur."
 
-#: internal_ops/views.py:244
+#: internal_ops/views.py:245
 msgid "There was an error sending the notification email to the requestor."
 msgstr ""
 "Une erreur s'est produite lors de l'envoi de l'e-mail de notification au "
 "demandeur."
 
-#: internal_ops/views.py:248 internal_ops/views.py:273
+#: internal_ops/views.py:249 internal_ops/views.py:274
 msgid "Please enter the information requested."
 msgstr "Veuillez saisir les informations demandées."
 
-#: internal_ops/views.py:265
+#: internal_ops/views.py:266
 msgid "We have successfully recorded the purchase information"
 msgstr "Nous avons enregistré avec succès les informations d'achat"
 
-#: internal_ops/views.py:270
+#: internal_ops/views.py:271
 msgid "There was an error recording the purchase information"
 msgstr ""
 "Une erreur s'est produite lors de l'enregistrement des informations d'achat"
 
-#: saas_app/views.py:24
+#: saas_app/views.py:27
 msgid "Something went wrong with the search. Please try again."
 msgstr "Une erreur s'est produite lors de la recherche. Veuillez réessayer."
 
-#: submit_request/forms.py:34
+#: saas_app/views.py:67
+msgid "You have successfully switched to {} role"
+msgstr "Vous êtes passé avec succès à {} rôle"
```

Comment:
We can check with translation services to confirm, but I think I would use something like this instead:
- Vous avez réussi à passer au rôle {}

I believe this syntax is more inclusive and gender neutral based on the training we had a few weeks ago.

Reply:
😃 I was trying to get this to work but for the life of me I can't translate the actual variable to french without making the code super ugly. I could not find a way in Django to do it easily because the variable value is unknown and Django refuses to translate it. BUT I will look into it some more since we should fix it.